src/utils/time.ts

```diff
@@ -27,14 +43,15 @@ export function addTimedelta(date: Date, components: DateTimeComponents) {
 }
 
 export function subtractTimedelta(date: Date, components: DateTimeComponents) {
-    return addTimedelta(date, negateTimeComponents(components));
+    return addTimedelta(date, negateDateTimeComponents(components));
 }
 
-export function negateTimeComponents(components: DateTimeComponents) {
-    const negComponents:TimeComponents = {}
-    for (const key in components) {
+export function negateDateTimeComponents(components: DateTimeComponents) {
```

Comment:
Why the API change here?

Reply:
Because the name led me to believe this would only negate the `hours, minutes, seconds, ms` properties, which it doesn't.
It's not just the properties from TimeComponents that are negated, every property of DateTimeComponents is negated (tested [here](https://github.com/T-Systems-MMS/aurelia-components/blob/master/tests/utils/time.test.ts#L38-L54)), so it should be named accordingly in my opinion.

Reply:
I prefer to just renamed it to `negateComponents`, so it depends on the passed data type.
Anyway: this is a breaking API change! You should at least add a wrapper method.

Reply:
Done.